### PR TITLE
Alt text

### PR DESCRIPTION
**Pull request by dphiffer**
## Overview

Front-end CSS for adding an alt text mechanism for images and videos on MLTSHP. It might make sense to consolidate discussion on the PR over on the mltshp repo: https://github.com/MLTSHP/mltshp/pull/710

## Screenshots

A screengrab of the alt text editor, with a textarea input, save button, cancel button, and a link to learn more about alt text.
A screengrab showing the visual alt text toggle below the image.

### CODE REVIEW

**Comment by dphiffer** [comment]
One issue I'm having with `yarn build` is that the resulting `main.min.css` doesn't include the inline SVG. What I do see in my web inspector is stuff like `background-image: svg-load("balloons/comment-quip-left.svg")`. So probably something is up with my postcss?

**Review by spaceninja — APPROVED** [review]

File: src/legacy/components/images/partials/_image-alt-text.scss [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    }
+
+    &.alt-text--hidden .the-alt-text,
+    &.alt-text--editing .the-alt-text {
+      display: none;
+    }
+  }
+
+  // Inline Editing Alt Text
+  .alt-text-edit .the-alt-text:hover,
```
Comment (spaceninja) [inline src/legacy/components/images/partials/_image-alt-text.scss:36]:
```suggestion
  // stylelint-disable-next-line a11y/selector-pseudo-class-focus this element is not focusable
  .alt-text-edit .the-alt-text:hover,
```

File: src/legacy/components/images/partials/_image-alt-text.scss [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    }
+
+    &.alt-text--hidden .the-alt-text,
+    &.alt-text--editing .the-alt-text {
+      display: none;
+    }
+  }
+
+  // Inline Editing Alt Text
+  // stylelint-disable-next-line a11y/selector-pseudo-class-focus this element is not focusable
```
Comment (spaceninja) [inline src/legacy/components/images/partials/_image-alt-text.scss:36]:
```suggestion
  // stylelint-disable-next-line a11y/selector-pseudo-class-focus
```

File: src/legacy/components/images/partials/_image-alt-text.scss [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+      display: none;
+    }
+  }
+
+  // Inline Editing Alt Text
+  // stylelint-disable-next-line a11y/selector-pseudo-class-focus
+  .alt-text-edit .the-alt-text:hover,
+  .the-alt-text-hover {
+    background-color: var(--color-status-edit);
+  }
```
Comment (spaceninja) [inline src/legacy/components/images/partials/_image-alt-text.scss:40]:
```suggestion
  // stylelint-disable a11y/selector-pseudo-class-focus
  .alt-text-edit .the-alt-text:hover,
  .the-alt-text-hover {
    background-color: var(--color-status-edit);
  }
  // stylelint-enable a11y/selector-pseudo-class-focus
```

**Pull request merged**